src/cascade/testing_utilities/fake_data.py

```diff
@@ -0,0 +1,82 @@
+from types import SimpleNamespace
+
+import numpy as np
+import pandas as pd
+
+from cascade.core import getLoggers
+from cascade.core.db import db_queries
+from cascade.executor.covariate_data import find_covariate_names
+from cascade.input_data.db.locations import get_descendants
+from cascade.input_data.db.locations import location_hierarchy
+from cascade.model.integrands import make_average_integrand_cases_from_gbd
+
+CODELOG, MATHLOG = getLoggers(__name__)
+
+
+def retrieve_fake_data(execution_context, local_settings, covariate_data_spec):
+    """Like :py:func:`cascade.executor.estimate_location` except makes
+    all fake data. This is deterministic."""
```

Comment:
It would be slightly more work to make this deterministic relative to a seed rather than hard coded. That seems useful...